cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/constants/WebServerConfig.java

```diff
@@ -334,6 +334,13 @@
       + "request, then the service user will be used as the authenticated principal (so there will be no delegation "
       + "or impersonation but rather a simple fallback to SPNEGO).";
 
+  /**
+   * <code>vertx.enabled</code>
+   */
+  public static final String VERTX_ENABLED_CONFIG = "vertx.enabled";
+  public static final boolean VERTX_ENABLED = false;
+  private static final String VERTX_ENABLED_DOC = "If Cruise Control should start with Vertx.io based API";
```

Comment:
Would be a bit nicer to say "Enables the Vertx.io based API and Swagger UI for Cruise Control".